test/associations.spec.js

```diff
@@ -0,0 +1,264 @@
+var should = require('should');
+var _ = require('lodash');
+var RSVP = require('rsvp');
+var request = require('supertest');
+var Promise = RSVP.Promise;
+var uuid = require('node-uuid');
+
+var seeder = require('./seeder.js');
+
+
+describe('associations', function () {
+
+    var config;
+    beforeEach(function () {
+        config = this.config;
+        return seeder(this.app).dropCollectionsAndSeed().then(function (_ids) {
```

Comment:
If you know what collections are used, would be nice to see them called out here, for speed and so that it becomes the default way people interact with your code.